.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: node_js
-sudo: false
-node_js:
-  - 0.10
-  - 0.12
+dist: trusty
+sudo: required
```

Comment:
Looks like it's specifying the Ubuntu image to use in the Travis env. It's likely required because of the version of Node it installs in the environment.